### ISSUE TITLE
Not able to add another Y axis in line plot

### ISSUE THREAD

**Issue by JafferWilson** [facebookresearch/visdom#713]
Hello @JackUrb ,
I have got another issue.
Here is the code that I am trying.
```
import visdom
import numpy as np
import json
from win32api import GetSystemMetrics
import time
# viz = visdom.Visdom()
viz = None
def Init_Visualization():
    global viz
    try:
        viz = visdom.Visdom()
        viz.delete_env("testing")
        viz.env = "testing"
        return True
    except Exception as e:
        print(str(e))
        return False
    
def Clear_Graphs():
    global viz
    data = viz.get_window_data()
    if len(data) >0:
        for i in json.loads(data):
            viz.close(i)
    # driver = webdriver.Chrome()
    # driver.get("http://localhost:8097")
def VizPlot(plt):   
    global viz
    viz.matplot(plt,opts=dict(resizable=True),win="plot")
def Draw_Lines(X,Y1,Y2,title="nothing",win="nothing",total =3):
    global viz
    viz.line(
    X=X,
    Y=Y1,
    win=win,
    name='0',
    opts=dict(
    title=title,
    legend=["orig","pred"],
    width = (GetSystemMetrics(0)/total)-50,
    height=300,
    )
    )
    viz.line(
    X=X,
    Y=Y2,
    win=win,
    name="pred",
    update="append"
    )
    time.sleep(3)
    
def Update_Error_Correction(train_error = 0, test_error=0):
    global viz
    viz.text("Train RMSE: {}   Test RMSE: {}".format(train_error,test_error),win="Error Correction",opts={'width':GetSystemMetrics(0),'height':10})
    
def Update_Line(X,Y,win="nothing"):
    global viz
    viz.line(X=None,Y=None,win=win,name="pred",update="remove")
    viz.line(
    X=X,
    Y=Y,
    win=win,
    name="pred",
    update="append"
    )
```
Using the above code, I have received the following image:
![newplot (1)](https://user-images.githubusercontent.com/13446197/77416909-8dfa7e00-6dea-11ea-80ab-a305bc516f6b.png)

Please let me know how I can add another Y-axis so that the data will get displayed properly.
I hope to hear from you soon.

**Comment by JafferWilson** [comment]
You see that the prediction output is not properly displayed as the scales are different for the orginal and prediction of the model.

**Comment by JackUrb** [comment]
@ubaidsworld implemented plots with two y axes in #719, which should allow you to do this (though it doesn't support appending at the moment, so if you want to have this plot update over time you'll need to either replace it repeatedly when you get new data or extend the implementation).

**Comment by ubaidsworld** [comment]
@JackUrb I will work on the update part later for sure.
